.changeset/fuzzy-ears-hide.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I think we should add a changeset for this. 😄

Reply:
There was a changeset for the feature, and we have not announced it yet. I think it might just be confusing.